packages/rocketchat-ui/client/lib/sideNav.coffee

```diff
@@ -36,8 +36,18 @@
 			sideNav.find("header").removeClass "hover"
 
 	focusInput = ->
+		sideNavDivs = _.filter document.querySelectorAll('aside.side-nav')[0].children, (ele) ->
+			ele.tagName == 'DIV' and !ele.classList.contains('hidden')
+		highestZidx = 0
+		highestZidxElem = undefined
+		_.each sideNavDivs, (ele) ->
+			zIndex = window.getComputedStyle(ele).zIndex
+			if Number(zIndex).toString() != 'NaN' and Number(zIndex) > highestZidx
```

Comment:
I think you can set `zIndex = Number(window.getComputedStyle(ele).zIndex)` and in this if just check `if zIndex > highestZidx`. NaN is false :) so NaN > any number will be false